src/Mantasflowers.Domain/Entities/UserContactInfo.cs

```diff
@@ -0,0 +1,9 @@
+namespace Mantasflowers.Domain.Entities
+{
+    public class UserContactInfo : BaseEntity
+    {
+        public string Email { get; set; }
+
+        public string Phone { get; set; }
```

Comment:
Email field should be nullable.

Reply:
Where will you send invoices, "temporary passwords", etc then?

Reply:
I was thinking about the unregistered users. Is it necessary to have both fields? Flower shop might be off with phone number as a contact measure, rather than asking for email as a requirement. Temporary passwords could be shown as is, right after submitting an order for the customer to mark it down.